app/models/comment.rb

```diff
@@ -1,4 +1,10 @@
 class Comment < ApplicationRecord
-  belongs_to :user
   belongs_to :post
+  belongs_to :user
+
+  after_save :updates_comments_counter
+
+  def updates_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
[OPTIONAL]
- [ ] Nice work implementing the `updates_comments_counter` method and using the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) **`after_save`** :clap: However it would be nice to make the `updates_comments_counter` private, since it will be call every time that a new comment has been save in the database.

```rb
after_save :updates_comments_counter
...
def updates_comments_counter
   post.update(comments_counter: post.comments.count)
end
```